pkg/config/derived-role-cert.go

```diff
@@ -43,29 +43,44 @@ func (idCfg *IdentityConfig) derivedRoleCertConfig() error {
 		return nil // disabled
 	}
 
-	if idCfg.roleCertDir == "" {
+	if idCfg.roleCertDir == "" && idCfg.roleCertNamingFormat == "" {
 		return nil // disabled
 	}
+	// If both the RoleCert settings and the NamingFormat settings are configured redundantly, an error will be returned.
+	if idCfg.roleCertDir != "" && idCfg.roleCertNamingFormat != "" {
+		return fmt.Errorf("RoleCertDir and RoleCertNamingFormat are both set: RoleCertDir %s, RoleCertNamingFormat %s", idCfg.roleCertDir, idCfg.roleCertNamingFormat)
```

Comment:
- Kinda redundant (rolecertdir and rolecertnamingformat appears twice)
- maybe follow SNAKE_CASE for env?
- let users know the reason what they are supposed to do to fix it: https://developers.google.com/tech-writing/error-messages/show-fix

```suggestion
		return fmt.Errorf("Both ROLECERT_DIR[%s] and ROLE_CERT_NAMING_FORMAT[%s] are set. Please ensure only one of these is specified to avoid conflicts.", idCfg.roleCertDir, idCfg.roleCertNamingFormat)
```

Reply:
handled in: https://github.com/AthenZ/k8s-athenz-sia/pull/149/commits/8515d06e7f4342ddf4c18ab94b97774fa107e598